docs/production/video-calls.md

```diff
@@ -22,8 +22,9 @@ installation, you'll need to register a custom Zoom app as follows:
 
 1. Inside the Zoom app management page:
 
-   - On the **App Credentials** tab, set both the **Redirect URL for
-     OAuth** and the **Whitelist URL** to
+   - On the **App Credentials** tab, set the **Redirect URL for
+     OAuth** to `htps://zulip.example.com/help/start-a-call` and the
+     **Whitelist URL** to
```

Comment:
It's currently `https://zulipchat.com/json/calls/register`. I'll also note we have both /zoom/register and /zoom/complete configured on the whitelist at present.

I think I agree that's wrong -- if I understand it correctly, the claim is that the "Redirect URL" is what happens when you try to add the Zulip app from the Zoom site?

Reply:
Hmm, what currently causes that to send you to `start-a-call`?